test/unit/integration/narrow/app/server.test.js

```diff
@@ -1,4 +1,26 @@
-import { server } from '../../../../../app/server.js'
+import { createHash } from 'crypto'
+import { hashEmail, server } from '../../../../../app/server.js'
+
+describe('hashEmail', () => {
+  it('should hash an email consistently', () => {
+    const email = 'Test@Example.com '
+    const expectedHash = createHash('sha256').update('test@example.com').digest('hex')
+    expect(hashEmail(email)).toBe(expectedHash)
+  })
```

Comment:
A clearer equivalence test might be just to call the function twice:
```suggestion
    expect(hashEmail(email)).toBe(hashEmail(email))
  })
```